verify_rpms/rpm_verifier.py

```diff
@@ -87,10 +91,21 @@ def get_unsigned_rpms(rpms: list[str]) -> list[str]:
         for rpm in rpms
         if "Key ID" not in rpm and not rpm.startswith("gpg-pubkey")
     ]
-
     return unsigned_rpms
 
 
+def get_signed_rpms_keys(rpms: list[str]) -> list[str]:
+    """
+    Get the keys used to sign RPMs
+    :param rpms: list of RPMs
+    :return: list of keys used to sign RPMs
+    """
+    signed_rpms_keys: list[str] = [
+        rpm.split(", Key ID ")[-1] for rpm in rpms if "Key ID" in rpm
+    ]
```

Comment:
I think you can replace the square brackets of curly ones and then you'd have a `set` and eliminate duplicates. The output will look cleaner I think:
```suggestion
    signed_rpms_keys: list[str] = {
        rpm.split(", Key ID ")[-1] for rpm in rpms if "Key ID" in rpm
    }
```

Reply:
We want the duplication as we count the number of times the key appears and add it to the json.

`results["keys"] = dict(Counter(processed_image.signed_rpms_keys).most_common())`

Reply:
oh. Understood.